cfgov/v1/util/filterable_list.py

```diff
@@ -116,3 +115,9 @@ def form_id(self):
             return form_ids[0]
         else:
             return 0
+
+    def serve(self, request):
```

Comment:
I think in that example case they don't pass `*args` or `**kwargs` because that particular page class doesn't do anything with those optional arguments, but the base `Page` method does support them. We also might not use them on `FilterableList` pages but in theory we could if we used [RouteablePageMixin](http://docs.wagtail.io/en/v1.9/reference/contrib/routablepage.html) somewhere. It'd be nice to support that option.